packages/EasyCodingStandard/packages/ChangedFilesDetector/src/config/services.yml

```diff
@@ -6,4 +6,6 @@ services:
 
     Symplify\EasyCodingStandard\ChangedFilesDetector\:
         resource: '..'
-        exclude: '../{Cache/CacheFactory}'
+
+    Nette\Caching\Cache:
+        factory: ['@Symplify\EasyCodingStandard\ChangedFilesDetector\Cache\CacheFactory', 'create']
```

Comment:
Better here than manually in every class

Reply:
Do you have some comparison with Symfony/Cahe? https://symfony.com/doc/current/components/cache.html What is the selling point of Nette Cache (I don't know, never actually used it).

Reply:
Not really, just used to it. I never used Symfony\Cache
Feel free to replace it though, it might be slight improvement in understandability as more standard.